status/projects.py

```diff
@@ -156,19 +149,12 @@ def project_summary_data(self, row):
             row.value["pending_reviews"] = links
 
         # Find the latest running note, return it as a separate field
-        if "running_notes" in row.value:
-            try:
-                notes = json.loads(row.value["running_notes"])
-                # note_dates = {datetime obj: time string, ...}
-                note_dates = dict(
-                    zip(map(dateutil.parser.parse, notes.keys()), notes.keys())
-                )
-                latest_date = note_dates[max(list(note_dates))]
-                row.value["latest_running_note"] = json.dumps(
-                    {latest_date: notes[latest_date]}
-                )
-            except ValueError:
-                pass
+        # TODO: Refactor to leverage new view and try get all running notes in a single call
+        latest_running_note = LatestRunningNoteHandler.get_latest_running_note(
+            self.application, "project", row.key[1]
```

Comment:
Same comment here. Could you perhaps use the couch possibility of fetching multiple keys at once?

Reply:
Thats why I made the TODO, didn't want to change too many things at once during the switch. It will require a bigger reordering of getting data for the lists. I could also make a Trello card.